### DISCUSSION TITLE
Command to work on different file endings?

### DISCUSSION THREAD

**Discussion by arkhandir** [ImageMagick/ImageMagick#6438] [Help] [answered]
### ImageMagick version

7.1.1-12

### Operating system, version and so on

macOS 12.6.7

### Description

How can I make the following command work both on files ending with *.tif and *.tiff?

> magick mogrify -format jpg *.tiff

Thanks!

**Comment by snibgo** [comment, marked as answer]
```
magick mogrify -format jpg *.tif*
```

**Reply by arkhandir** [reply]
Thanks, this should have been obvious. 

What about *.tif and *.jpg?

**Comment by snibgo** [comment]
> What about *.tif and *.jpg?

I don't understand. Why do you want to convert jpg files to jpg?

You can use a list for the filenames. For example:
```
magick mogrify -format jpg *.png s*.tif*
```
This will make jpg files from all files named `*.png` and `s*.tif*`.

**Reply by arkhandir** [reply]
Sorry, this was a rather stupid example. I just wanted to know how to operate with input files with completely different file extensions. Thank you.
